src/components/PaletteSwitcherDialog/index.tsx

```diff
@@ -69,7 +69,7 @@ const PaletteSwitcherDialog = ({ dialogOpen, handleClose }: Props) => {
       buttonName="Close"
     >
       <FormControl>
-        <FormLabel id="palette-mode-group-label">Color Mode</FormLabel>
+        <FormLabel id="palette-mode-group-label"></FormLabel>
```

Comment:
```suggestion
```
If the goal is to remove the redundancy of the text, simply remove the whole element.